### ISSUE TITLE
Would it be a valid solution?

### ISSUE THREAD

**Issue by sdoshi983** [incubateind/gdtc-Day2#34]
Dear @kroy96,

As a part of following the steps given in the readme file, I cloned the repo and ran the code in my local system. On running I found problems with margin and padding at some places. So would it be wise to land a PR for such small bugs? Would it be considered as a valid PR?

Regards

**Comment by rexyagami** [comment]
Yes, this is the task for you to either fix all the given issues in the design, or you can fix a few and generate the PR. The PR will be considered as a valid event if you fixed small bugs or issues.

**Comment by sdoshi983** [comment]
I landed the PR fixing more than one issue.
Thank you!
